datasetUtils/getUserInfoFromToken.go

```diff
@@ -54,12 +54,12 @@ func GetUserInfoFromToken(client *http.Client, APIServer string, token string) (
 	if err != nil {
 		return map[string]string{}, []string{}, err
 	}
-	req2, err := http.NewRequest("GET", APIServer+"/users/"+url.QueryEscape(newUserInfo.Id)+"/userIdentity", nil)
```

Comment:
This endpoint seems to still exist (at least in release-jobs). What's the issue here?